jbohde/pull-request-demo
# Add README text

## To test

- Open `index.html` and press the increment button
- Verify that the count updates by 1 and the UI is updated as well.
